Mw/all

michaelwood opened the pull request.

michaelwood commented:
(rebased)

Bjwebb commented on cove_oc4ids/templates/cove_oc4ids/additional_checks_table.html:
```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+{% load cove_tags %}
+
+<table class="table">
+    <thead>
+      <tr>
+        <th>{% trans 'Check Description' %}</th>
+        <th>{% trans 'Location of first 3 errors' %}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for type, values in additional_checks.items %}
+        {% if type == 'empty_field' %}
+          <tr>
+            <td>{% trans 'The data includes fields that are empty or contain only whitespaces. Fields that are not being used, or that have no value, should be excluded in their entirety (key and value) from the data' %}</td>
+            <td>
+              <ul class="list-unstyled">
+                {% for value in values|slice:":3" %}
+                  <li>{{ value.json_location }}</li>
+                {% endfor %}
+                {% if values|length > 3 %}
+                  <li><a data-toggle="modal" data-target=".{{"additional-checks-"|concat:forloop.counter}}"><sup>see all</sup></a></li>
+                {% endif %}
+              </ul>
+            </td>
+          </tr>
+        {% endif %}
+      {% endfor %}
+    </tbody>
+</table>
+
+{% for type, values in additional_checks.items %}
+  {% cove_modal_errors className="additional-checks-"|concat:forloop.counter modalTitle="" errorList=values|list_from_attribute:"json_location" file_type=file_type full_table=False %}
+{% endfor %}
```
Is it worth trying to pull this file in from the cove-ocds repo, rather than copy/pasting it?

michaelwood replied:
@Bjwebb I did have a think a long those lines ...but for 34 lines of pretty basic template I couldn't see a non-complicated way of doing that? Did you have any suggestions?  cove-ocds isn't packaged as a python package or really as a django application, maybe a git submodule and a symlink might work, but a bit fragile. In an ideal world I think I would have made relevant parts of cove/lib-cove-web as django applications so that things like template inheritance could work, but that is slightly out of scope!

jpmckinney replied:
Maybe just leave a comment at the top of the file to link to its source, then.

michaelwood commented:
Updated the PR, changed:

- Added project and resource name to tx config (now that we have them)
- Re-reset translations
- Added comment to additional checks table to note the original source of the file
